### ISSUE TITLE
Failed fetching btoa-lite dependency (since 2.6.0)

### ISSUE THREAD

**Issue by tovbinm** [MunifTanjim/node-bitbucket#91]
package-lock.json

```
 "node_modules/@node-fetch/btoa-lite": {
      "version": "0.0.0",
      "resolved": "git+ssh://git@github.com/node-fetch/btoa-lite.git#1b038dc61d790ca68495a754f5b656dea01f7d63",
      "license": "MIT",
      "engines": {
        "node": ">=10"
      }
    },
```

**Comment by MunifTanjim** [comment]
Which package manager? Which command is failing? What was the error?

**Comment by tovbinm** [comment]
```
 npm ERR! Error while executing:
npm ERR! /usr/bin/git ls-remote -h -t ssh://git@github.com/node-fetch/btoa-lite.git
npm ERR! 
npm ERR! Warning: Permanently added the RSA host key for IP address '192.30.255.113' to the list of known hosts.
npm ERR! git@github.com: Permission denied (publickey).
npm ERR! fatal: Could not read from remote repository.
npm ERR! 
npm ERR! Please make sure you have the correct access rights
npm ERR! and the repository exists.
npm ERR! 
npm ERR! exited with error code: 128
```

**Comment by MunifTanjim** [comment]
Released `v2.6.1`. That should fix this.

**Comment by tovbinm** [comment]
Thanks!! Works.
